drivers/spi/spi_ll_stm32.c

```diff
@@ -704,7 +694,7 @@ static int spi_stm32_release(const struct device *dev,
 
 #if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)
 static int32_t spi_stm32_count_bufset_frames(const struct spi_config *config,
-	                                     const struct spi_buf_set *bufs)
+	const struct spi_buf_set *bufs)
```

Comment:
Identation was fine before

Reply:
As explained in https://github.com/zephyrproject-rtos/zephyr/pull/66503#discussion_r1426785103, that makes the compliance check to fail.

Reply:
You can observe yourself that the indentation you propose is not in line with indentation that can be seen otherwise in this file and most of other zephyr c files.
Whatever issue you faced previously, I think the error was elsewhere.
Also, note that if you don't change a line, it won't be inspected by scripts and should not trigger any error.

Reply:
I think the problem was that I had tab size of 4 characters before, which led me to add too many tabs and cause an error. As a consequence, I only wanted to add one tab and then all spaces, which causes another error. The test passes now.